src/style.css

```diff
@@ -1 +1,217 @@
-/* empty source */
+* {
+  margin: 0;
+}
+
+.card {
+  width: 198px;
```

Comment:
- it definitely should not be 198px if we're asked for 200px maximum
- try to use variables